metadata.tvshows.themoviedb.org.python/resources/language/resource.language.en_gb/strings.po

```diff
@@ -74,7 +74,11 @@ msgctxt "#30108"
 msgid "Enable character art"
 msgstr ""
 
-# empty strings from 30109 to 30149
+msgctxt "#30109"
```

Comment:
Was this missing before or did you forget to include the settings.xml update?

Reply:
Crap. That's a bad merge from the Matrix branch. And that bad merge affects some code too. Let me fix that and do another commit. Thanks for catching that.

Reply:
Fixed with c9d5938